### ISSUE TITLE
worktree equivalent

### ISSUE THREAD

**Issue by colemickens** [martinvonz/jj#41]
Does this currently have a `git worktree` equivalent? Or can it be used with git worktrees somehow?

**Comment by martinvonz** [comment]
Yes, that's actually already kind of supported if you use the Git backend. For example, you can set up a Git repo with two addition working copies managed by jj by doing something like this:
```
git init git && (cd git && echo a > file && git add file; git ci -m initial)
jj init --git-store=git jj1
jj init --git-store=git jj2
```

There's issue #13 about adding more native support for it. The support described above means that each `jj1` and `jj2` has its own view of which anonymous heads and branches there are in the repo, for example. You can still check out any commit from either directory, but I want to make them share the set of heads and branches (and the history in the "operation log"). I plan to make it so you can see in the `jj log` output where each working copy's checkout is.

What's your use case? Do you have an existing Git repo and want to try using `jj` in a new working copy?

**Comment by martinvonz** [comment]
I just closed issue #13, meaning that the native support for multiple working copies is now done. See [my comment at the end](https://github.com/martinvonz/jj/issues/13#issuecomment-1028634355) to read about how it works.
